core/ecschema-metadata/src/SchemaJsonLocater.ts

```diff
@@ -23,7 +24,21 @@ export type SchemaPropsGetter = (schemaName: string) => SchemaProps | undefined;
  * @public @preview
  */
 export class SchemaJsonLocater implements ISchemaLocater {
-  public constructor(private _getSchema: SchemaPropsGetter) { }
+  private _getSchema: SchemaPropsGetter;
+
+  public constructor(schemaPropschemaGetter: SchemaPropsGetter) {
```

Comment:
Something seems off with that name

Reply:
yeah, a bit too much schema in it, I changed the argument name to getSchemaProps